From c94f4cf4d5a6a14f2e65c44a01aa3fcc9b0c1261 Mon Sep 17 00:00:00 2001
From: Stefan Assmann <sassmann@redhat.com>
Date: Tue, 26 Aug 2014 14:12:07 -0400
Subject: [ethernet] igb: Cleanups to fix braces location warnings

Message-id: <1409062359-13063-50-git-send-email-sassmann@redhat.com>
Patchwork-id: 89170
O-Subject: [RHEL7.1 PATCH 49/81] igb: Cleanups to fix braces location warnings
Bugzilla: 1091121
RH-Acked-by: Nikolay Aleksandrov <nikolay@redhat.com>
RH-Acked-by: Neil Horman <nhorman@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Carolyn Wyborny <carolyn.wyborny@intel.com>

This patch fixes WARNING:BRACES and ERROR:OPEN_BRACE from
checkpatch file check.

Signed-off-by: Carolyn Wyborny <carolyn.wyborny@intel.com>
Tested-by: Aaron Brown <aaron.f.brown@intel.com>
Signed-off-by: Jeff Kirsher <jeffrey.t.kirsher@intel.com>
Signed-off-by: Stefan Assmann <sassmann@redhat.com>
(cherry picked from commit d34a15abfe370252de83e14e763cf7fcb8c84585)
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/intel/igb/e1000_82575.c b/drivers/net/ethernet/intel/igb/e1000_82575.c
index 7b7d08c..b1a759c 100644
--- a/drivers/net/ethernet/intel/igb/e1000_82575.c
+++ b/drivers/net/ethernet/intel/igb/e1000_82575.c
@@ -73,9 +73,8 @@ static s32  igb_validate_nvm_checksum_82580(struct e1000_hw *hw);
 static s32  igb_update_nvm_checksum_82580(struct e1000_hw *hw);
 static s32 igb_validate_nvm_checksum_i350(struct e1000_hw *hw);
 static s32 igb_update_nvm_checksum_i350(struct e1000_hw *hw);
-static const u16 e1000_82580_rxpbs_table[] =
- { 36, 72, 144, 1, 2, 4, 8, 16,
-   35, 70, 140 };
+static const u16 e1000_82580_rxpbs_table[] = {
+ 36, 72, 144, 1, 2, 4, 8, 16, 35, 70, 140 };
 
 /**
  *  igb_sgmii_uses_mdio_82575 - Determine if I2C pins are for external MDIO
@@ -1436,9 +1435,8 @@ static s32 igb_reset_hw_82575(struct e1000_hw *hw)
 
  /* set the completion timeout for interface */
  ret_val = igb_set_pcie_completion_timeout(hw);
- if (ret_val) {
+ if (ret_val)
   hw_dbg("PCI-E Set completion timeout has failed.\n");
- }
 
  hw_dbg("Masking off all interrupts\n");
  wr32(E1000_IMC, 0xffffffff);
diff --git a/drivers/net/ethernet/intel/igb/igb_ethtool.c b/drivers/net/ethernet/intel/igb/igb_ethtool.c
index 33c289d..f772bbe 100644
--- a/drivers/net/ethernet/intel/igb/igb_ethtool.c
+++ b/drivers/net/ethernet/intel/igb/igb_ethtool.c
@@ -1196,8 +1196,8 @@ static bool reg_pattern_test(struct igb_adapter *adapter, u64 *data,
 {
  struct e1000_hw *hw = &adapter->hw;
  u32 pat, val;
- static const u32 _test[] =
-  {0x5A5A5A5A, 0xA5A5A5A5, 0x00000000, 0xFFFFFFFF};
+ static const u32 _test[] = {
+  0x5A5A5A5A, 0xA5A5A5A5, 0x00000000, 0xFFFFFFFF};
  for (pat = 0; pat < ARRAY_SIZE(_test); pat++) {
   wr32(reg, (_test[pat] & write));
   val = rd32(reg) & mask;
diff --git a/drivers/net/ethernet/intel/igb/igb_main.c b/drivers/net/ethernet/intel/igb/igb_main.c
index bff1a91..cdd6c87 100644
--- a/drivers/net/ethernet/intel/igb/igb_main.c
+++ b/drivers/net/ethernet/intel/igb/igb_main.c
@@ -4235,9 +4235,8 @@ static void igb_watchdog_task(struct work_struct *work)
 
    /* check for thermal sensor event */
    if (igb_thermal_sensor_event(hw,
-       E1000_THSTAT_LINK_THROTTLE)) {
+       E1000_THSTAT_LINK_THROTTLE))
     netdev_info(netdev, "The network adapter link speed was downshifted because it overheated\n");
-   }
 
    /* adjust timeout factor according to speed/duplex */
    adapter->tx_timeout_factor = 1;
@@ -4473,13 +4472,12 @@ static void igb_update_itr(struct igb_q_vector *q_vector,
  case low_latency:  /* 50 usec aka 20000 ints/s */
   if (bytes > 10000) {
    /* this if handles the TSO accounting */
-   if (bytes/packets > 8000) {
+   if (bytes/packets > 8000)
     itrval = bulk_latency;
-   } else if ((packets < 10) || ((bytes/packets) > 1200)) {
+   else if ((packets < 10) || ((bytes/packets) > 1200))
     itrval = bulk_latency;
-   } else if ((packets > 35)) {
+   else if ((packets > 35))
     itrval = lowest_latency;
-   }
   } else if (bytes/packets > 2000) {
    itrval = bulk_latency;
   } else if (packets <= 2 && bytes < 512) {
-- 
1.7.1